lib/build-plugin.js

```diff
@@ -66,4 +102,4 @@ function buildPlugin(Client, pluginOpts) {
   return plugin
 }
 
-module.exports = buildPlugin
+module.exports = { buildPlugin }
```

Comment:
any reason for this change?

Reply:
Originally thought I may need to export some of the factored out functions to meet coverage requirements, turned out to be a non-issue